src/routes/inngang/inngang-hovedkategorier.tsx

```diff
@@ -14,16 +14,23 @@ import {
     WhiteBackgroundContainer
 } from '../../styled-components/common';
 import Layout from '../layout';
+import { UseDocumentTitle } from '../../page-title/page-title';
 
 const InngangHovedkategorier = () => {
+    const title = 'Klage eller anke på vedtak';
     useLogPageView(PageIdentifier.INNGANG_HOVEDKATEGORIER);
+    useEffect(() => {
```

Comment:
I stedet for å gjenta `useEffect` med samme callback på alle sider, så kan vi skille det ut i en "custom" hook.
```ts
export function usePageInit(title: string) {
    useEffect(() => {
         document.title = title;
         window.scrollTo(0, 0);
    }, [title]);
}
```